app/lib/katello/resources/candlepin/owner.rb

```diff
@@ -18,7 +18,7 @@ def create(key, description, content_access_mode: 'org_environment')
               :displayName => description,
               :contentPrefix => "/#{key}/$env",
               :contentAccessMode => content_access_mode,
-              :contentAccessModeList => ['entitlement', 'org_environment'].join(',')
+              :contentAccessModeList => 'org_environment'
```

Comment:
(non-blocking) since the :contentAccessModeList is now just a single item, should we create a card to simplify and refactor in the future? or does this construction come from candlepin and changes would need to be made on their first?

let me know what you think @jeremylenz

Reply:
Candlepin takes a comma-separated list, which was previously `"entitlement,org_environment"` but is now simply `"org_environment"`. So I don't see any refactoring that could be done on our end, unless I'm missing something?

Reply:
No, that answered my question. Thanks!